compile-server.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
Bash is not guaranteed to be present on Windows, so this package can only work with Linux and macOS, right?

Reply:
This script is just for us to update server dependency. It's not part of released package.

Reply:
In what way is this "compilation" when it's just downloading, unzipping and copying some files around? Can't we do this step in the installation step of a LanguageHandler/AbstractPlugin?

Reply:
Yeah, it's not compiling. Not even building. Just extracts some files from the released artifacts.

We only need to do that when we want to update the server to a newer version.
We have similar scripts in some other pacakges.

I don't want to have version update on its own for users as that can lead to changes that could break the package.

We wouldn't need it if they would release it as a standalone language server but no response in my question there.

Reply:
I see. So it's a thing that must be done at develop-time, not at runtime.